README.md

```diff
@@ -375,6 +375,11 @@ The semi-official [webcomponents/custom-elements] polyfill is what GitHub uses,
 and it provides a bunch of workarounds for the spec rules involving class
 constructors and the `new` keyword. You should use it, too!
 
+Unfortunately the custom element polyfill provided by `@webcomponents` does not support extended built-in custom elements, has this [known limitations](https://github.com/webcomponents/custom-elements#known-bugs-and-limitations) and [**is slow**](https://github.com/webcomponents/custom-elements#customelementspolyfillwrapflushcallback).
```

Comment:
I would object to this wording. The "slow" link references a feature to deviate from spec-compliance for better performance. There's no evidence given here that other polyfills are faster, or that they've even considered this case. If they do synchronous upgrade, then they're just as "slow". If the do async upgrade, they're not spec compliant. If the have an option, then they're equivalent to the `@webcomponents` polyfill. Basically, this is hearsay.

Also, while the `@webcomponents` polyfills do indeed have a detailed known issues section, that doesn't mean that other polyfills are without their known issues (a fully faithful polyfill might not be the right choice in all cases after all). Indeed, in the text below there's already a "constructor caveat" which, honestly, is a _huge_ problem if I read it correctly. It seems to mean that most standard custom elements won't work because they won't have their constructor written in such a specific way. There's also a "caveats" section: https://github.com/WebReflection/document-register-element#common-issues--caveat Pointing out the `@webcomponts` known issues without pointing out the others comes across as disingenuous.

Reply:
@justinfagnani 
Thanks for your opinion.

I'm sorry that you feel this is hearsay. Please correct me, but the link I attached clearly states:
> The polyfill gets **slower** as the size of your page and number of custom element definitons increases. You can use polyfillWrapFlushCallback to prevent redundant work.

You are right, also WebReflection's polyfills comes with caveats.
That's why I mentioned them too, as you can see below:
> please note the `constructor` caveat ...

My purpose here is to be objective and show the good and bad parts clearly.

May I ask, do I touch work of yours?
I mean do I criticize work you where involved?